internal/app/urlstorage/databasestorage.go

```diff
@@ -158,7 +157,13 @@ func (s *DatabaseStorage) DeleteUserURLs(ctx context.Context, urlsByUser ...URLs
 
 // Clear all mappings.
 func (s *DatabaseStorage) Clear() error {
-	return errors.New("not implemented")
+	tx, err := s.DB.BeginTx(context.Background(), nil)
+	if err != nil {
+		return err
```

Comment:
К ошибкам лучше добавлять контекст, что-то типа:

```suggestion
		return fmt.Errorf("failed to begin a transcation: %w", err)
```

Reply:
Да, буду пользоваться!